.github/workflows/behat-stability-1-basic-installation.yml

```diff
@@ -14,7 +14,8 @@ env:
 jobs:
   # This workflow contains a single job called "Running tests"
   tests:
-    name:  Basic Installation
+    name:  'Behat Basic Installation'
+    if: ${{ !github.event.pull_request.draft }}
```

Comment:
Read for review above makes this check redundant.

Reply:
Unfortunately not, the "ready_to_review" will trigger the check when the PR changes from draft -> ready-to-review, but it will also trigger on every commit due to the "sync" event that I added above. This check will ensure to only check commits when the PR is not on "draft" state